### DISCUSSION TITLE
Laragon.org is blocked due to malware

### DISCUSSION THREAD

**Discussion by mark1502** [leokhoa/laragon#722] [General]
Is there malware on laragon.org?  I tried to view the site and was blocked by two different AV guards, so I avoided it.

I guess I can just visit github, but just wondering about laragon.org

**Comment by leokhoa** [comment]
https://laragon.org is a static website.

**Reply by mark1502** [reply]
Thanks for your reply.  More importantly, thank you for Laragon, it's excellent!

It's good to know the laragon.org is just a static site, so it's probably better to go to the github if any questions arise.

Not sure why the browsers stopped me from visiting laragon.org.  Maybe some files got hacked there, or maybe the url is just on a malware list.

Fortunately, the github is available.  You know, without Laragon, I may have given up on Laravel before learning about all of its great features.  Thanks again!

**Reply by sambedingfield** [reply]
Just to confirm that this is an issue I'm also facing, Malwarebytes is currently blocking laragon.org as a phishing site.

![image](https://github.com/leokhoa/laragon/assets/13186243/01b4c16b-766c-4a8a-a121-fb9f27aec16e)

This causes an `ERR_SSL_PROTOCOL_ERROR` error in Chrome when visiting the website.

![image](https://github.com/leokhoa/laragon/assets/13186243/529ba216-ff1a-404c-b57f-512e83a55e34)
Adding an exclusion then restarting Chrome fixes the issue.

I'd also like to confirm that yes, Laragon is absolutely excellent!
You should advertise your GitHub sponsorship on the repo! 
Go [Sponsor Leo](https://github.com/sponsors/leokhoa)!!! ❤️
